src/Illuminate/Database/Grammar.php

```diff
@@ -63,6 +63,14 @@ public function wrap($value, $prefixAlias = false)
             }
 
             return $this->wrap($segments[0]).' as '.$this->wrapValue($segments[2]);
+        } elseif (strpos(strtolower($value), ' ') !== false) {
+            $segments = explode(' ', $value);
+
+            if ($prefixAlias) {
+                $segments[1] = $this->tablePrefix.$segments[1];
+            }
+
+            return $this->wrap($segments[0]).' as '.$this->wrapValue($segments[1]);
         }
```

Comment:
@taylorotwell sorry forgot to remove `as` on this line. Pushing another commit to fix this. Thanks for reviewing the PR.